onboard/package.json

```diff
@@ -60,6 +60,7 @@
     "react-test-renderer": "^17.0.2",
     "reselect": "^4.0.0",
     "styled-components": "^5.3.1",
+    "superagent": "^8.0.0",
```

Comment:
Is library 'superagent' needed and being used? I'm not sure I saw it anywhere in the implementation.

Reply:
Deleted

Reply:
This has not been deleted yet.